controller/src/test/java/io/pravega/controller/server/v1/ZKControllerServiceImplTest.java

```diff
@@ -238,4 +238,10 @@ public void getCurrentSegmentsKeyValueTableTest() {
         //TODO: Place holder till we fix the test
         assertTrue(true);
     }
+
+    @Test
+    public void kvtablesInScopeTest() {
+        //TODO: Place holder till we fix the test
```

Comment:
When are we fixing the test?

Reply:
As discussed, these are tests for Zookeeper based implementation of KVTables and I won't be fixing them in this PR.